relay-general/src/store/normalize.rs

```diff
@@ -118,28 +118,41 @@ impl<'a> NormalizeProcessor<'a> {
     }
 
     /// Validates the timestamp range and sets a default value.
-    fn normalize_timestamps(&self, event: &mut Event) -> ProcessingResult {
+    fn normalize_timestamps(
+        &self,
+        event: &mut Event,
+        meta: &mut Meta,
+        state: &ProcessingState<'_>,
+    ) -> ProcessingResult {
         let received_at = self.config.received_at.unwrap_or_else(Utc::now);
         event.received = Annotated::new(received_at);
 
+        let mut sent_at = None;
+
         event.timestamp.apply(|timestamp, meta| {
             if let Some(secs) = self.config.max_secs_in_future {
                 if *timestamp > received_at + Duration::seconds(secs) {
                     meta.add_error(ErrorKind::FutureTimestamp);
-                    return Err(ProcessingAction::DeleteValueSoft);
+                    sent_at = Some(*timestamp);
+                    return Ok(());
                 }
             }
 
             if let Some(secs) = self.config.max_secs_in_past {
                 if *timestamp < received_at - Duration::seconds(secs) {
                     meta.add_error(ErrorKind::PastTimestamp);
-                    return Err(ProcessingAction::DeleteValueSoft);
+                    sent_at = Some(*timestamp);
+                    return Ok(());
                 }
             }
 
             Ok(())
         })?;
 
+        ClockDriftProcessor::new(sent_at, received_at)
+            .reason("Timestamp out of bounds.")
```

Comment:
Our error messages are usually lower case.

```suggestion
            .reason("timestamp out of bounds")
```